### PR TITLE
[295] Create a reusable heading component 

### PR DESCRIPTION
**Pull request by snowiesuet**
### PR Checklist:

Hey there! It's great to have you here. Here's a quick checklist to help you make a shiny PR:

* [x] Have you checked out the guidelines in our [Contributing](https://github.com/FightPandemics/FightPandemics/blob/master/CONTRIBUTING.md) document?
* [x] Have you looked if there might be other open [Pull Requests](https://github.com/FightPandemics/FightPandemics/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) for the same update/change?

### Quick Description of the PR
We had a few different headings component and all of them have different styles. We should standardize them based on the design styleguide (Design Systems MVP).

### An Overview of the Changes

- Replaced `CustomH1`, `Title/Title`, `Typography/Title` components with `Heading `and `TextLabel`

- There should be no reason to modify the style of the fonts as they are currently following the Design System MVP Typography.

- Props that `Heading` accept are color and level (1, 2, 3, 4)

- Props that `TextLabel` accept color, font-size (should be taken from theme.typography.font), font-weight and block (`TextLabel` is a span)

### CODE REVIEW

File: client/src/components/Typography/Heading.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+const Heading = styled(Title)`
+  &.ant-typography {
+    font-weight: bold;
+    color: ${(props) => props.color};
+    ${theme.typography.heading.font}
+  }
+
+  &.h1 {
+    font-size: 4.2rem;
```
Comment (anniegiang) [inline client/src/components/Typography/Heading.js:17]: For all the font-sizes let's use them from `theme.js`

**Comment by silenceway** [comment]
Thinking in the accessibility, Heading should return a header tag (h1, h2, h3, etc) instead of a class with the type.

**Comment by anniegiang** [comment]
> Thinking in the accessibility, Heading should return a header tag (h1, h2, h3, etc) instead of a class with the type. @anniegiang: What do you think?

Good point. Does the `Title` component return a class instead of a heading element? If so, we may need to rethink this.

**Comment by snowiesuet** [comment]
> Thinking in the accessibility, Heading should return a header tag (h1, h2, h3, etc) instead of a class with the type. @anniegiang: What do you think?

They are returning in h1, h2, h3, h4 but I couldn't find a way to edit the default font-size directly, so I added the class.

**Comment by anniegiang** [comment]
> > Thinking in the accessibility, Heading should return a header tag (h1, h2, h3, etc) instead of a class with the type. @anniegiang: What do you think?
> 
> They are returning in h1, h2, h3, h4 but I couldn't find a way to edit the default font-size directly, so I added the class.

Overriding ant styles can be tricky, most of the time the selector needs to be very specific in order to override. But I think the current implementation works as well.

**Pull request merged**